govcd/vdc.go

```diff
@@ -310,6 +310,13 @@ func (vdc *Vdc) ComposeVApp(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate
 	if vapptemplate.VAppTemplate.Children == nil || orgvdcnetworks == nil {
 		return Task{}, fmt.Errorf("can't compose a new vApp, objects passed are not valid")
 	}
+
+	primaryNetworkConnectionIndex := 0
```

Comment:
Please add a comment above this block explaining what it's doing and what primaryNetworkConnectionIndex really means.

Reply:
Awesome!